spec/models/plant_spec.rb

```diff
@@ -0,0 +1,7 @@
+require 'rails_helper'
+
+RSpec.describe Plant do
+  describe "relationships" do
+    it {should have_many :plots}
```

Comment:
Be sure to test: `it { should have_many(:plots).through(:plot_plants) }` and `it { should have_many :plot_plants } ` to be more precise about the way plot and plant are connected

The same associations should be added to plot_spec as well